docs/en/building/supplemental.md

```diff
@@ -1,3 +1,43 @@
+# Application Base Path
+
+All of your applications links, images, and resources are served from your applications base path. By default, the base path is `/`, but you can configure the base path by adding the `base` option to your `.dojorc`.
+
+> .dojorc
+
+```json
+{
+	"build-app": {
+		"base": "./some-directory/"
+	}
+}
+```
+
+## Not Hosted from Root
+
+You may need to change your base path if your Dojo app is not served from the root of your web server. For example, if your app is served from `http://example.com/incredible-app`, you would update your base path to be `/incredible-app/`.
```

Comment:
```suggestion
If a Dojo app is not served from the root of the web server, it may be necessary to change the base path . For example, if an app is served from `http://example.com/incredible-app`, update the base path to be `/incredible-app/`.
```